### PR TITLE
Added Paracosma Nepal

### PR DESCRIPTION
**Pull request by pinological**
Added paracosma-nepal.json with paracosma detail.

### CODE REVIEW

File: data/paracosma-nepal.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Paracosma Nepal",
+    "establishedAt": 1451589361,
+    "location": "naxal, Kathmandu",
```
Comment (Saphall) [inline data/paracosma-nepal.json:4]: As per https://www.rojgarisanjal.com/company/detail/paracosma-470 : `Nagpokhari Marg, Gairidhara, Kathmandu`

File: data/paracosma-nepal.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "website": "https://www.paracosma.com/",
+    "email": null,
```
Comment (Saphall) [inline data/paracosma-nepal.json:6]: Email according to : https://www.rojgarisanjal.com/company/detail/paracosma-470 : `info@paracosma.com`

File: data/paracosma-nepal.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    "phone": "+977-1-4004778",
```
Comment (Saphall) [inline data/paracosma-nepal.json:7]:
```suggestion
    "phone": "+977-14004778",
```

**Review by Saphall — CHANGES_REQUESTED** [review]

**Comment by thenishantsapkota** [comment]
Please make all the required changes as previously requested. @pinological

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]
Please make all the required changes as previously requested. @pinological

File: data/paracosma-nepal.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    "socials": {
+      "facebook": "https://www.facebook.com/paracosmaInc",
+      "instagram": "https://www.instagram.com/paracosma_studio/",
+      "github": null,
+      "linkedin": "https://www.linkedin.com/company/paracosma/",
+      "twitter": "https://twitter.com/Paracosmainc",
+      "youtube": "https://www.youtube.com/channel/UCf4_ArJPd3vT2FmErBbEcUA"
+    }
+  }
+
```
Comment (Saphall) [inline data/paracosma-nepal.json:17]: There seems some formatting issue at the end of the line. 👀 Please refer to other json files once and adjust any spaces or new lines accordingly. 🙏

**Pull request closed without merging**